state-processor/src/main/java/com/ververica/flink/training/exercises/ReadRidesAndFaresSnapshot.java

```diff
@@ -0,0 +1,66 @@
+package com.ververica.flink.training.exercises;
+
+import org.apache.flink.api.common.state.ValueState;
+import org.apache.flink.api.common.state.ValueStateDescriptor;
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.runtime.state.memory.MemoryStateBackend;
+import org.apache.flink.state.api.ExistingSavepoint;
+import org.apache.flink.state.api.Savepoint;
+import org.apache.flink.state.api.functions.KeyedStateReaderFunction;
+import org.apache.flink.training.exercises.common.datatypes.TaxiFare;
+import org.apache.flink.training.exercises.common.datatypes.TaxiRide;
+import org.apache.flink.util.Collector;
+
+/**
+ * Java implementation for an example using the State Processor API to read and display
+ * the contents of a retained checkpoint or savepoint from RidesAndFaresSolution.
+ */
+public class ReadRidesAndFaresSnapshot {
+
+	/***************************************************************************************
+	 Update this path to point to a checkpoint or savepoint from RidesAndFaresSolution
```

Comment:
This comment fails on checkstyle `./gradlew clean check shadowJar`, but if you make this configurable, then it shouldn't be a problem anymore.

Reply:
Yep.